client/src/components/PacmanGame/PacmanBoard.jsx

```diff
@@ -14,9 +13,17 @@ function PacmanBoard({ gridState, gridSize }) {
   if (gridState === '') {
     return null;
   }
-  console.log(gridSize);
-  console.log(codeToEntity(gridState[43][5]));
   let game = [];
+
+  const flattenArray = (inputArray) => {
```

Comment:
You can use Array.prototype.flat() instead.

Reply:
https://jsperf.com/flatten-an-array-loop-vs-reduce/38
 Loop works better than _reduce_ and _prototype.flat()_ while flattening the array.

Reply:
Array.flat() completed first.

Reply:
Read the x-axis. It's the operations per second.